packages/cloud-cognitive/src/components/Datagrid/Datagrid.stories.js

```diff
@@ -36,6 +36,9 @@ import {
   useCustomizeColumns,
   useSelectAllWithToggle,
 } from '.';
+
+import useColumnCenterAlign from './useColumnCenterAlign';
```

Comment:
You should import this hook with all the other hooks above (not from it's own import).